src/main/java/org/owasp/validator/html/scan/MagicSAXFilter.java

```diff
@@ -66,20 +68,24 @@
 	private ResourceBundle messages;
 
 	private boolean isNofollowAnchors;
+	private boolean isNoopenerAndNoreferrerAnchors;
 	private boolean isValidateParamAsEmbed;
 	private boolean inCdata = false;
     // From policy
     private boolean preserveComments;
     private int maxInputSize;
     private boolean shouldParseImportedStyles;
 
+	private static final Logger logger = LoggerFactory.getLogger(MagicSAXFilter.class);
+
     public MagicSAXFilter(ResourceBundle messages) {
 		this.messages = messages;
     }
 
     public void reset(InternalPolicy instance){
         this.policy = instance;
         isNofollowAnchors = policy.isNofollowAnchors();
+		isNoopenerAndNoreferrerAnchors = policy.isNoopenerAndNoreferrerAnchors();
```

Comment:
This line has a different type of white space in front. Please change to match the previous/following lines.